content/en/tracing/profiling/search_profiles.md

```diff
@@ -0,0 +1,138 @@
+| Runtime Info | Text dump of all the runtime properties                                                                                                     |
+
+**Note**: In the upper right corner of each profile, there are options to:
+
+- Download the profile
+- Switch the profile to full screen
+
+
+### Profile types
+
+In the **Profiles** tab you can see all profile types available for a given language. Depending on the language, the information collected about your profile differs.
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+{{< img src="tracing/profiling/profile.png" alt="A specific profile">}}
+
+Once enabled, the following profile types are collected:
+
+| Profile type             | Definition                                                                                                                                                                                                                                                                                         |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU in Java Code         | Shows the time each method spent running on the CPU. It includes JVM bytecode, but not native code called from within the JVM.                                                                                                                                                                     |
+| Allocation               | Shows the amount of heap memory allocated by each method, including allocations which were subsequently freed.                                                                                                                                                                                     |
+| Wall Time in Native Code | Shows the elapsed time spent in native code. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the method is running. This profile does not include time spent running JVM bytecode, which is typically most of your application code. |
+| Class load               | Shows the number of classes loaded by each method.                                                                                                                                                                                                                                                 |
+| Error                    | Shows the number of errors thrown by each method.                                                                                                                                                                                                                                                  |
```

Comment:
With the new exception sampling changes, this should likely be renamed to Throwables or something like that.

Reply:
What should we put instead? `Exceptions Thrown`?

Reply:
It should probably be singular to match the other types. They're like that as a ~shorthand of "X Profile", eg "Allocation Profile".

`Exception` isn't technically precise but miiight be useful because of it's user-friendliness. Alex's suggestion of `Throwable` is the more accurate version.

In either case we should say in the description that it includes Exceptions and Errors.